.travis-tox.ini

```diff
@@ -58,6 +58,7 @@ deps =
     setuptools
     wheel
     py36,py37,py38: numpy
+    py36,py37,py38: mypy
```

Comment:
Minor, but maybe move this one line up so the two numpy entries remain adjacent.

Reply:
I added a separate section, as requested